### PR TITLE
Update ResourceManager versionto 2.2.0-preview

### PR DESCRIPTION
**Pull request by hjhhh3000vitae**
## Description

Update the Resource Manager SDK version to 2.2.0-preview.

Related SDK PR: https://github.com/Azure/azure-sdk-for-net/pull/6840

## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by hjhhh3000vitae** [comment]
@erich-wang Could you please review this PR?

File: src/Resources/Resources/ChangeLog.md [hunk 1]
```diff
@@ -19,7 +19,8 @@
 -->
 ## Upcoming Release
 - Remove missing cmdlet referenced in `New-AzResourceGroupDeployment` documentation
-- Updated policy cmdlets to use new api version 2019-01-01
+- Update policy cmdlets to use new api version 2019-01-01
+- Add support for new api version 2019-05-10
```
Comment (markcowl) [inline src/Resources/Resources/ChangeLog.md]: What is the actual, practical impact of thsi change from the customer perspecitve?  What new capabilities are you exposing in cmdlets?

Comment (hjhhh3000vitae) [reply src/Resources/Resources/ChangeLog.md]: ChangeLog updated. Please review.

Comment (hjhhh3000vitae) [reply src/Resources/Resources/ChangeLog.md]: @markcowl Could you please review this PR? I have updated it as requested.

**Comment by cormacpayne** [comment]
@Tiano2017 Hey Tian, would you mind taking a look at this PR? I want to make sure it doesn't impact the work you're doing with the deployment cmdlets before we merge.

File: src/Resources/Resources/ChangeLog.md [hunk 2]
```diff
@@ -18,6 +18,9 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+* Add support for new api version 2019-05-10 for `New-AzResourceGroupDeployment`
```
Comment (Tiano2017) [inline src/Resources/Resources/ChangeLog.md]: nit: I think the API version change is for all cmdlets.

**Comment by Tiano2017** [comment]
@cormacpayne thanks for the heads up. The PR looks good to me. I'll just need to merge this PR to mine.

File: src/Resources/Resources/ChangeLog.md (hunk 2 above)
Comment (hjhhh3000vitae) [reply src/Resources/Resources/ChangeLog.md]: Truely. I have updated it. Thanks.

**Comment by hjhhh3000vitae** [comment]
@cormacpayne @markcowl Could you please review this PR? Remarks have be resolved.

**Pull request merged**